test/functional/core/main_spec.lua

```diff
@@ -194,3 +194,41 @@ describe('command-line option', function()
     matches('Compilation: .*Run :checkhealth', fn.system({ nvim_prog_abs(), '-V1', '-v' }))
   end)
 end)
+
+if is_os('win') then
+  describe('Windows features', function()
+    it('expands ~/', function()
```

Comment:
this it() can live in the describe() block starting on line 17. Else this is missing before_all(...).

```suggestion
    it('expands ~/ on Windows', function()
```